version

```diff
@@ -1 +1 @@
-5.0.0
\ No newline at end of file
+5.0.1
```

Comment:
it is a feature, so you need to bump minor, not patch.
since it's an alpha feature, we don't have to follow semver completely (e.g. breaking changes don't case major version bump), but at the same time it's not fair to call something a patch release when it's not